packages/react-native/template/ios/HelloWorld/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
```

Comment:
Duplicating this info here defeats the purpose of having it at the library level no? As far as I understand, all these manifests gets merged into one.

Reply:
hi unfortunately the required reasons of the manifests do not get aggregated, only the data use permissions